src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import tasks.*;
+import io.Message;
+import io.Parser;
+
+import java.util.Scanner;
+
 public class Duke {
+
+    static final int MAX_TASKS = 100;
+
+    static Task[] tasks = new Task[MAX_TASKS];
+    static int taskCount = 0;
+
+    public static void addTask(Task task){
```

Comment:
consider standardizing the format of brackets across all the files, like while (true){ vs while(true) { and while (true) {
use egyptian style brackets as mentioned in the course website